build_scripts/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+ENV_FILE=.env
```

Comment:
Can we remove `ENV_FILE` here and in the other script files, and reference `.env` directly instead? Also, I think we should error if `.env` does not exist since it should be part of a working setup

Reply:
I'm doing this for local development start scripts, but I think the build scripts need to be able to handle those env varibales just being set in the environment by salt so the .env should be optional there.

Reply:
Makes sense